Src/Microsoft.Dynamic/Debugging/CollectionUtils.cs

```diff
@@ -30,16 +30,24 @@ internal static class CollectionUtils {
                 return false;
             }
             var cmp = EqualityComparer<T>.Default;
-            var f = first.GetEnumerator();
-            var s = second.GetEnumerator();
-            while (f.MoveNext()) {
-                s.MoveNext();
 
-                if (!cmp.Equals(f.Current, s.Current)) {
-                    return false;
+            IEnumerator<T> f = null;
+            try {
```

Comment:
Why use a try/finally for the first enumerator instead of two usings?

Reply:
I had https://msdn.microsoft.com/en-us/library/ms182334.aspx in mind but looking at this again we can use two usings since they are independent of each other.